Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
I added all packages from the wiki + pytest and pycodestyle + flake8 code conventions. Note that I only needed numpy and matplotlib to run the python script.